generators/heroku/index.js

```diff
@@ -566,112 +658,83 @@ module.exports = class extends BaseGenerator {
-                                                                            )
-                                                                        );
-                                                                        this.log(
-                                                                            chalk.yellow(
-                                                                                `And you can view the logs with this command\n\t${chalk.bold(
-                                                                                    'heroku logs --tail'
-                                                                                )}`
-                                                                            )
-                                                                        );
-                                                                        this.log(
-                                                                            chalk.yellow(
-                                                                                `After application modification, redeploy it with\n\t${chalk.bold(
-                                                                                    'jhipster heroku'
-                                                                                )}`
-                                                                            )
-                                                                        );
-                                                                    }
-                                                                    done();
-                                                                }
-                                                            );
-                                                            child.stderr.on('data', data => {
-                                                                const line = data.toString().trimRight();
-                                                                if (line.trim().length !== 0) this.log(line);
-                                                            });
-                                                        }
-                                                    );
-                                                }
-                                            );
-                                        }
-                                    );
-                                }
-                            });
+
+                    try {
+                        const gitAdd = execCmd(gitAddCmd);
+                        gitAdd.child.stdout.on('data', data => {
+                            this.log(data);
+                        });
+
+                        gitAdd.child.stderr.on('data', data => {
+                            this.log(data);
+                        });
+                        await gitAdd;
+
+                        this.log(chalk.cyan(gitCommitCmd));
```

Comment:
You should defined the command above this, it will be easier to read.
Otherwise we must go back to the start of the task.